webrtc-stats.html

```diff
@@ -3463,6 +3474,40 @@ <h2>
           </section>
         </div>
       </section>
+      <section id="sctptransportstats-dict*">
+        <h3>
+          <dfn>RTCSctpTransportStats</dfn> dictionary
+        </h3>
+        <p>
+          An <code>RTCSctpTransportStats</code> object represents the stats
+          corresponding to an <code><a>RTCSctpTransport</a></code> described in
+          [[!WEBRTC]].
+        </p>
+        <div>
+          <pre class="idl">dictionary RTCSctpTransportStats : RTCStats {
+    double smoothedRoundTripTime;
+};</pre>
+          <section>
+            <h2>
+              Dictionary <a class="idlType">RTCSctpTransportStats</a> Members
+            </h2>
+            <dl data-link-for="RTCSctpTransportStats" data-dfn-for="RTCSctpTransportStats" class=
+            "dictionary-members">
+              <dt>
+                <dfn><code>smoothedRoundTripTime</code></dfn> of type
+                <span class="idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  The latest smoothed round-trip time value, corresponding to
+                  spinfo_srtt defined in [[RFC6458]] but converted to seconds.
```

Comment:
@vr000m When you review can you take a look at the relevant RFCs if there is something else that we could reference from the public SCTP API?

Reply:
https://tools.ietf.org/html/rfc4960#section-6.3.1 gives the calculation of SRTT.

Reply:
This will report the instantaneous value of SRTT when getStats is called.

Reply:
@henbos 
sorry missed the mention.
converted to seconds means it'll be a fracture?
as in `25`ms will be converted to `0.025`s ?